build-utils/sentry-instrumentation.js

```diff
@@ -8,24 +8,24 @@ const {
   NODE_ENV,
   SENTRY_INSTRUMENTATION,
   SENTRY_WEBPACK_WEBHOOK_SECRET,
-  TRAVIS_COMMIT,
-  TRAVIS_BRANCH,
-  TRAVIS_PULL_REQUEST,
-  TRAVIS_PULL_REQUEST_BRANCH,
+  GITHUB_SHA,
+  GITHUB_REF,
+  CI,
 } = process.env;
-const IS_CI = !!TRAVIS_COMMIT;
+const IS_CI = !!CI;
```

Comment:
No. Please use a more specific identifier to do this rather than just checking CI. See https://github.com/getsentry/sentry/pull/21415/ and related bug for more context around this.

Reply:
@BYK FWIW it's only enabled when `SENTRY_INSTRUMENTATION` is configured: https://github.com/getsentry/sentry/pull/22034/files#diff-bb490b7bd0a004bce840fd8ce6b94911de263f1eb455491f30596ea4a0aee48eR29

`IS_CI` is used for Sentry tags to see if we are in a CI env.

Reply:
Sure but I've got burned by this before. Maybe we should rename it or something?